11326-add-select-photo

rodkot opened the pull request:
Добавил PhotoSelectorFragment, в котором открывается галерея по контракту PickImageContract и выбранные фотографии сохраняются  в PhotoController.selectedPhotos, затем фрагмент закрывается

rodkot committed:
Добавил PhotoSelectorFragment, в котором открывается галерея и выбранные фотографии сохраняются  в PhotoController.selectedPhotos

DoHKapJleoHe commented on app/src/main/java/ru/nsu/ccfit/tsd/pinmap/photo/Photo.kt:
```diff
@@ -0,0 +1,7 @@
+
+import android.net.Uri
+
+data class Photo (
```
Не уверен, что этот класс не будет конфликтовать с тем, что будет у Наташи

rodkot committed:
Удалил:
- фрагмент выбора из галереи
- контроллер фотографий
Добавил:
= контракт выбора фотографий по URI
- отображение фоток по URI в конструктуре Pins
Debug:
- странное отображение карточек фотографий

v-kharchenko reviewed (approved):
Вынести в задачи:
1. При добавлении фотографий старые не должны удаляться.
2. Удаление фотографий по нажатию.
3. Записывать URI фотографий в сохраняемый Pin.
4. При добавлении фотографий должны быть видны фотографии, которые уже были выбраны.
